crates/goose-server/src/commands/mcp.rs

```diff
@@ -14,6 +15,15 @@ pub async fn run(name: &str) -> Result<()> {
     }
 
     tracing::info!("Starting MCP server");
+
+    if name == "rmcp_developer" {
```

Comment:
there's some similar mcp startup code in goose-cli (we should de-duplicate that at some point)